ui/helpers/constants/design-system.js

```diff
@@ -127,9 +127,11 @@ export const DISPLAY = {
   FLEX: 'flex',
   GRID: 'grid',
   INLINE_BLOCK: 'inline-block',
+  INLINE: 'inline',
   INLINE_FLEX: 'inline-flex',
   INLINE_GRID: 'inline-grid',
   LIST_ITEM: 'list-item',
+  NONE: 'none',
 };
```

Comment:
Now that we have responsive props it makes sense to add `none` to the display types. There should be many use cases for showing/hiding components at different screen sizes.